agent/Dockerfile

```diff
@@ -1,7 +1,14 @@
 FROM debian:10.3-slim
```

Comment:
Huh - I wonder why we're not using Ubuntu here? Nothing against Debian, but we're using Ubuntu pretty much everywhere else and it seems to be the best-supported by NVIDIA. I suspect they have more resources to be keeping Ubuntu base images, etc. up to date too. I wonder if we should switch.

Reply:
I was suggesting earlier we could at least use `debian:10-slim`, since it's up to date already.

Reply:
But @ioga mentioned some potential troubles with the same approach in envs.

Reply:
we had the same issues with nvidia base images not having security upgrades couple months before. newer tags also didn't matter/help for nvidia. they just didn't update them on time.

Reply:
yep, confirmed latest release of `debian:10-slim` doesn't have the lz4 patch, so we'd need to do this anyway.

Reply:
ah but 10.10 does, I guess 10 != 10.10, which I assumed. oops.